src/content/translations/fr/governance/index.md

```diff
@@ -0,0 +1,184 @@
+
+<Divider />
+
+## Développement de la chaîne phare {#beacon-chain}
+
+Le processus de gouvernance Ethereum négocie souvent la vitesse et l'efficacité pour l'ouverture et l'inclusivité. Afin d'accélérer le développement de la chaîne phare, elle fût lancée séparément du réseau Ethereum par preuve de travail et a suivi ses propres pratiques de gouvernance.
+
+Bien que le développement des spécifications et des implémentations ait toujours été entièrement open source, les processus formels utilisés pour proposer des mises à jour décrites ci-dessus n'ont pas été utilisés. Cela a permis de préciser les changements et de parvenir plus rapidmement à un accord entre chercheurs et développeurs.
+
+Lorsque la chaîne phare fusionnera avec la couche d'exécution Ethereum, le processus de gouvernance pour proposer des changements sera harmonisé. Ce processus d'implémentation de la fusion est [déjà en cours](https://github.com/ethereum/EIPs/pull/3675).
+
+<ButtonLink to="/upgrades/merge/">
+  Plus d'infos sur la fusion
+</ButtonLink>
+
+<Divider />
+
+## Comment s'impliquer ? {#get-involved}
+
+- [Proposer une EIP](/eips/#participate)
+- [Discuter des propositions actuelles](https://ethereum-magicians.org/)
+- [Impliquez-vous dans la discussion R&D](https://ethresear.ch/)
+- [Rejoignez le Discord Ethereum R&D](https://discord.gg/mncqtgVSVw)
+- [Exécuter un nœud](/developers/docs/nodes-and-clients/run-a-node/)
+- [Contribuer au développement du client](/developers/docs/nodes-and-clients/#execution-clients)
+- [Programme d’apprentissage pour les développeurs de base](https://blog.ethereum.org/2021/09/06/core-dev-apprenticeship-second-cohort/)
+
+## Complément d'information {#further-reading}
+
+La gouvernance d'Ethereum n'est pas rigoureusement définie. Divers participants de la communauté ont des points de vue différents à ce sujet. Voici quelques-unes d'entre elles :
+
+- [Notes on Blockchain Governance](https://vitalik.ca/general/2017/12/17/voting.html) - _Vitalik Buterin_
+- [Governance on Ethereum](https://docs.ethhub.io/ethereum-basics/governance/) – _ETHHub_
+- [How does Ethereum Governance work?](https://cryptotesters.com/blog/ethereum-governance) – _Cryptotesters_
+- [How Ethereum governance works](https://medium.com/coinmonks/how-ethereum-governance-works-71856426b63a) – _Micah Zoltu_
+- [What is an Ethereum core developer?](https://hudsonjameson.com/2020-06-22-what-is-an-ethereum-core-developer/) - _Hudson Jameson_
+- [Governance, Part 2: Plutocracy Is Still Bad](https://vitalik.ca/general/2018/03/28/plutocracy.html) - _Vitalik Buterin_
+- [Moving beyond coin voting governance](https://vitalik.ca/general/2021/08/16/voting3.html) - _Vitalik Buterin_
```

Comment:
I think we decided that we **would** translate external article titles? @lukassim 

<img width="1002" alt="Screenshot 2022-02-24 at 14 55 24" src="https://user-images.githubusercontent.com/62268199/155548519-3094fa4a-0ead-4371-8a0f-3f7508553c4e.png">


https://ethereumorgwebsitedev01-frusecases.gtsb.io/fr/governance